multisrc/overrides/madara/falconmangaar/src/FalconMangaAr.kt

```diff
@@ -0,0 +1,8 @@
+/* ktlint-disable */
+// THIS FILE IS AUTO-GENERATED; DO NOT EDIT
+package eu.kanade.tachiyomi.extension.ar.falconmangaar
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+
+
+class FalconMangaAr : Madara("فالكون مانجا", "https://falconmanga.com/", "ar")
```

Comment:
```suggestion
class FalconMangaAr : Madara("فالكون مانجا", "https://falconmanga.com", "ar")
```

No need for trailling "/"